src/Controller/ArtworkController.java

```diff
@@ -2,30 +2,42 @@
 
 import Model.ArtWork;
 
-import java.util.Collections;
 import java.util.LinkedHashMap;
-import java.util.List;
+import java.util.Scanner;
 
 public class ArtworkController {
     private LinkedHashMap<String, ArtWork> artworkList;
 
     private static ArtworkController artworkListInstance = null;
 
     private ArtworkController (){
+        this.artworkList = new LinkedHashMap<>();
+    }
 
-       /* artworkList = new LinkedHashMap<>();
-        ArtWork monaLisa =new ArtWork("Mona Lisa","Mysterious painting of a woman", "Leonardo Da Vinci", 1948, "Realism");
-        ArtWork sunRise = new ArtWork("Sunrise","Painting of boats and a sunrise", "Claude Monet", 1950, "Impressionism");
-
-        this.artworkList.put("Mona Lisa", monaLisa);
-        this.artworkList.put("Sunrise", sunRise);
-
-        */
+    public void createArtwork(){
+        InventoryController inventoryInstance = InventoryController.getInstance();
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Artwork name: ");
+        String artworkName = scanner.next();
+        System.out.println("Artwork Description: ");
+        String artworkDescription = scanner.next();
+        System.out.println("Artwork author: ");
+        String artworkAuthor = scanner.next();
+        System.out.println("Artwork adquisition year: ");
+        int artworkAdquisitionYear = scanner.nextInt();
+        System.out.println("Artwork artstyle: ");
+        String artworkArtStyle = scanner.next();
+
+
+        ArtWork artWork = new ArtWork(artworkName,artworkDescription,artworkAuthor,artworkAdquisitionYear,artworkArtStyle,inventoryInstance.getInventoryList().get("Inventory 1"));
+        this.artworkList.put(artWork.getName(),artWork);
```

Comment:
for all these methods that are interactive can u add a success/failure message?

Reply:
Done